load_db_from_json_file_pgsql.sh

```
@@ -0,0 +1,4 @@
+python manage.py flush --no-input 
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)